src/style.css

```diff
@@ -0,0 +1,32 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  background-color: white;
+  position: relative;
+}
+
+.yellow {
+  display: flex;
+  justify-content: flex-start;
+}
```

Comment:
do you need this class? can you move these styles to the main `stars__star` class?